### PR TITLE
fix(storages-container): add return to container option

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-9093`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-9664
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/containers/container/container.html [hunk 1]
```diff
@@ -18,6 +18,18 @@
...
         </oui-guide-menu>
     </oui-header>
+    <oui-header
+        heading="{{ :: $ctrl.container.name }}"
+        data-ng-if="!$ctrl.archive"
+    ></oui-header>
+    <!--return to conatiner-->
+    <oui-back-button
+        data-ng-if="!$ctrl.archive"
+        aria-label="Aria label for heading"
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/containers/container/container.html:28]: **nitpick:** Provide a proper label

Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/containers/container/container.html:28]: **nitpick:** Provide a proper label.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/containers/container/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "pci_projects_project_storages_containers_container_back_label": "Retour",
+  "pci_projects_project_storages_containers_container_back_button_label": "mes conteneurs",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/containers/container/translations/Messages_fr_FR.json:3]:
```suggestion
  "pci_projects_project_storages_containers_container_back_button_label": "Mes conteneurs",
```

File: packages/manager/modules/pci/src/projects/project/storages/containers/container/container.html (hunk 1 above)
Comment (sachinrameshn) [reply packages/manager/modules/pci/src/projects/project/storages/containers/container/container.html:28]: removed aria-label property and added data-previous-page property for back button.

File: packages/manager/modules/pci/src/projects/project/storages/containers/container/translations/Messages_fr_FR.json (hunk 2 above)
Comment (sachinrameshn) [reply packages/manager/modules/pci/src/projects/project/storages/containers/container/translations/Messages_fr_FR.json:3]: Done

**Review by anooparveti — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**